methods/fetchAndSet.js

```diff
@@ -5,7 +5,8 @@ module.exports = function(path, key, allDone) {
     if (err) {
       return allDone(err);
     }
-    server.store.set(key, result);
+    const store = typeof result === 'object' ? JSON.stringify(result) : result.toString();
+    server.store.set(key, store);
```

Comment:
this means your store.get needs to parse as well.  We should move this logic to the plugin